schema/field.go

```diff
@@ -103,19 +104,20 @@ func (schema *Schema) ParseField(fieldStruct reflect.StructField) *Field {
 	)
 
 	field := &Field{
-		Name:                   fieldStruct.Name,
-		DBName:                 tagSetting["COLUMN"],
-		BindNames:              []string{fieldStruct.Name},
-		FieldType:              fieldStruct.Type,
-		IndirectFieldType:      fieldStruct.Type,
-		StructField:            fieldStruct,
-		Tag:                    fieldStruct.Tag,
-		TagSettings:            tagSetting,
-		Schema:                 schema,
-		Creatable:              true,
-		Updatable:              true,
-		Readable:               true,
-		PrimaryKey:             utils.CheckTruth(tagSetting["PRIMARYKEY"], tagSetting["PRIMARY_KEY"]),
+		Name:              fieldStruct.Name,
+		DBName:            tagSetting["COLUMN"],
+		BindNames:         []string{fieldStruct.Name},
+		FieldType:         fieldStruct.Type,
+		IndirectFieldType: fieldStruct.Type,
+		StructField:       fieldStruct,
+		Tag:               fieldStruct.Tag,
+		TagSettings:       tagSetting,
+		Schema:            schema,
+		Creatable:         true,
+		Updatable:         true,
+		Readable:          true,
+		PrimaryKey: utils.CheckTruth(tagSetting["PRIMARYKEY"], tagSetting["PRIMARY_KEY"],
+			tagSetting["PRIMARYKEY,PRIORITY"]),
```

Comment:
Thank you for your contribution, can we keep the style of tags consistent?

https://github.com/go-gorm/gorm/blob/master/schema/index_test.go#L16

Reply:
Do you mean that the key for `tagSetting` should be `PRIMARYKEY` and `PRIORITY` instead of `PRIMARY,PRIORITY` ?

Reply:
`gorm:"primaryKey:,priority:1"`

Reply:
I've updated the parsing logic according to the tag format. I've also added a comment for the default priority value.

Reply:
```go
PrimaryKey: utils.CheckTruth(tagSetting["PRIMARYKEY"], tagSetting["PRIMARY_KEY"],
			tagSetting["PRIMARYKEY,PRIORITY"])
```
`tagSetting["PRIMARYKEY,PRIORITY"]` Is it unnecessary?